### PR TITLE
Calculator v1

### PR DESCRIPTION
**Pull request by Jolita-byte**

### CODE REVIEW

File: .idea/Java-Homework.iml [hunk 1]
```diff
@@ -23,5 +24,17 @@
     <orderEntry type="library" scope="TEST" name="Maven: org.apiguardian:apiguardian-api:1.0.0" level="project" />
     <orderEntry type="library" scope="TEST" name="Maven: org.opentest4j:opentest4j:1.1.1" level="project" />
     <orderEntry type="library" scope="TEST" name="Maven: org.junit.platform:junit-platform-commons:1.4.2" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: junit:junit:4.12" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.hamcrest:hamcrest-core:1.3" level="project" />
+    <orderEntry type="library" name="Maven: org.assertj:assertj-core:3.16.1" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.mockito:mockito-core:3.4.4" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: net.bytebuddy:byte-buddy:1.10.13" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: net.bytebuddy:byte-buddy-agent:1.10.13" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.objenesis:objenesis:2.6" level="project" />
```
Comment (Almantask) [inline .idea/Java-Homework.iml:33]: Where did those come from 😄 ?

File: src/Calculator/Calculator.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package Calculator;
+
+public class Calculator {
+    private char[] defaultOperators = {'+', '-', 'x', '%', '/', '!'};
```
Comment (Almantask) [inline src/Calculator/Calculator.java]: Why default?

File: src/Calculator/Calculator.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        if (expression.isEmpty() || expression.equals("")) {
+            return 0;
+        }
+        charArray = expression.toCharArray();
+        int result = Integer.parseInt(String.valueOf(charArray[0]));
+        for (int i = 1; i < charArray.length; i = i + 2) {
+            if ((i + 1) != charArray.length) {
+                result = operation(result, charArray[i], Integer.parseInt(String.valueOf(charArray[i + 1])));
+            } else result = operation(result, charArray[i], 0);
+        }
```
Comment (Almantask) [inline src/Calculator/Calculator.java:18]: Looks exactly like https://github.com/Almantask/Java-Homework/pull/21/files 
Same issues as there...
Did you help each other out?

**Review by Almantask — CHANGES_REQUESTED** [review]

File: .idea/Java-Homework.iml (hunk 1 above)
Comment (Jolita-byte) [reply .idea/Java-Homework.iml:33]: I have no idea :(

File: src/Calculator/Calculator.java (hunk 3 above)
Comment (Jolita-byte) [reply src/Calculator/Calculator.java:18]: No, I did all by myself. Changed function, so it can calculate with many digits.

**Review by Almantask — APPROVED** [review]

**Pull request closed without merging**